pkg/virt-api/webhooks/mutating-webhook/mutators/migration-create-mutator.go

```diff
@@ -91,3 +89,7 @@ func addMigrationSelectorLabel(migration *v1.VirtualMachineInstanceMigration) {
 
 	migration.Labels[v1.MigrationSelectorLabel] = migration.Spec.VMIName
 }
+
+func addMigrationFinalizer(migration *v1.VirtualMachineInstanceMigration) {
+	migration.Finalizers = append(migration.Finalizers, v1.VirtualMachineInstanceMigrationFinalizer)
+}
```

Comment:
nit: I know it was like that before your PR, but are we sure the Finalizers is always initialized?
Perhaps it's worth adding something like:
```go
if migration.Finalizers == nil {
	migration.Finalizers = make(map[string]string)
}
```

Reply:
The finalizers are of type `[]string`.

Reply:
Its ok to append to a nil slice.
Please see https://go.dev/tour/moretypes/15

Reply:
Good point. You're right.
Thanks!